network-api/networkapi/templates/fragments/buyersguide/pni_footer.html

```diff
@@ -0,0 +1,14 @@
+{% extends "partials/footer.html" %}
+
+{% load i18n l10n static %}
+
+{% block general_links %}
+    <li class="tw-mb-4"><a id="donate-footer-btn" href="?form=donate&utm_medium=FMO&utm_source=PNI&utm_campaign=23-PNI&utm_content=footer&c_id=7014x000000eQOD" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Donate" %}</a></li>
+    <!-- the rest of the links should be listed alphabetically -->
+    <li class="tw-mb-4"><a href="https://careers.mozilla.org/listings/?team=Mozilla%20Foundation" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Careers" %}</a></li>
+    <li class="tw-mb-4"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Cookies" %}</a></li>
+    <li class="tw-mb-4"><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Legal" %}</a></li>
+    <li class="tw-mb-4"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Participation Guidelines" %}</a></li>
+    <li class="tw-mb-4"><a href="https://foundation.mozilla.org/en/press-center/" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Press Center" %}</a></li>
+    <li ><a href="https://mozilla.org/privacy/websites/" class="tw-dark tw-text-white hover:tw-text-blue-20 hover:tw-underline">{% trans "Privacy" %}</a></li>
+{% endblock %}
```

Comment:
```suggestion
{% endblock general_links %}
```

I see that there is a error in the wagtail CI test due to the linter returning a `T003` error which is "end blocks should include a name". 

This should fix it 👍